build.sbt

```diff
@@ -42,23 +43,25 @@ lazy val microservice = Project(name, file("."))
       "-language:postfixOps"
     )
   )
-  .settings(
-    // To resolve a bug with version 2.x.x of the scoverage plugin - https://github.com/sbt/sbt/issues/6997
-    libraryDependencySchemes ++= Seq("org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always)
-  )
+
 val name = "check-eori-number-api"
 val scoverageExcludePatterns = List(
   "<empty>",
   "Reverse.*",
-  "uk.gov.hmrc.*",
   "prod.*",
   "app.Routes.*",
+  ".*Routes.*",
   "config",
-  "testOnlyDoNotUseInAppConf",
-  "repositories.MongoErrorLogger",
-  "controllers.AgentDelegateAuthorisationController"
+  "uk.gov.hmrc.checkeorinumberapi.controllers.ApiDocumentationController",
+  ".*views.*"
```

Comment:
@tapiwa-tiyemba Do we need views here?

Reply:
@ashwinirudrappa The reason I added the “.*views.*” filter is because was scanning definition.scala.txt in the views directory